release.xml

```diff
@@ -58,8 +58,14 @@
 	</copy>
     </target>
 
+    <target name="copyLicense" unless="useEFTLicensefile">
```

Comment:
@bshannon I am trying to copy the LICENSE_EFTL.md to LICENSE.md. I see there is an issue with this copy. This needs correction. If you find anything please let me know.

All the other changes to generate  javamail eftl bundle is present in this PR.